docs/project-configuration/tutorials/work-with-multiple-repositories.mdx

```diff
@@ -0,0 +1,577 @@
+engine will only have access to your local subgraphs and their accompanying metadata, you'll need to define
+relationships using your knowledge of other subgraphs and then test them using a supergraph build in your hosted
+environment.
+
+```yaml title="Add the following relationship object to your Users.hml file in your customer-team repo:"
+---
+kind: Relationship
+version: v1
+definition:
+  name: orders
+  sourceType: Users
+  target:
+    model:
+      subgraph: billing
+      name: Orders
+      relationshipType: Array
+  mapping:
+    - source:
+        fieldPath:
+          - fieldName: id
+      target:
+        modelField:
+          - fieldName: userId
+```
+
+```sh title="Create a new subgraph build:"
+ddn subgraph build create
+```
+
+```sh title="Get the list of supergraph builds:"
+ddn supergraph build get
+```
+
+```sh title="Then, create a supergraph build incorporating your latest subgraph build:"
+ddn supergraph build create --subgraph-version customers:<build-version> --base-supergraph-version <supergraph-build-id>
+```
+
+```graphql title="Now, you can test your nested query across subgraphs:"
+query GET_USERS_AND_ORDERS {
+  customers_users {
+    id
+    name
+    orders {
+      id
+      createdAt
+      status
+    }
+  }
+}
+```
+
+```json title="With a repsonse like this:"
+{
+  "data": {
+    "customers_users": [
+      {
+        "id": "7cf0a66c-65b7-11ed-b904-fb49f034fbbb",
+        "name": "Sean",
+        "orders": [
+          {
+            "id": "7ff13435-b590-4d6b-957f-f7fd39d4528a",
+            "createdAt": "2023-10-29T17:02:50.958076+00:00",
+            "status": "complete"
+          }
+        ]
+      },
+      {
+        "id": "82001336-65b7-11ed-b905-7fa26a16d198",
+        "name": "Rob",
+        "orders": [
+          {
+            "id": "9891596a-a732-4c1c-902c-1a112da48fec",
+            "createdAt": "2023-10-29T17:02:51.150084+00:00",
+            "status": "complete"
+          }
+        ]
+      },
+      {
+        "id": "86d5fba0-65b7-11ed-b906-afb985970e2e",
+        "name": "Marion",
+        "orders": [
+          {
+            "id": "85581445-752a-4aef-9684-b648eb5d5f42",
+            "createdAt": "2023-10-29T17:02:51.085229+00:00",
+            "status": "complete"
+          }
+        ]
+      },
+      {
+        "id": "8dea1160-65b7-11ed-b907-e3c5123cb650",
+        "name": "Sandeep",
+        "orders": [
+          {
+            "id": "c7406b75-6b24-41e4-9c5b-ff3feada9447",
+            "createdAt": "2023-10-29T17:02:50.889261+00:00",
+            "status": "processing"
+          }
+        ]
+      },
+      {
+        "id": "9bd9d300-65b7-11ed-b908-571fef22d2ba",
+        "name": "Abby",
+        "orders": [
+          {
+            "id": "98612470-1feb-4b91-88f7-9289d652ee87",
+            "createdAt": "2023-10-29T17:02:51.021317+00:00",
+            "status": "complete"
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+## Recap
```

Comment:
Is this inconsistent on purpose? For the first two tutorials, the last section was "next steps" and for this one it's "recap".

Reply:
Yep...there is no logical next step for this one. So, I decided to treat it as a summary.